### ISSUE TITLE
Vérification de la configuration

### ISSUE THREAD

**Issue by joelclems** [PnX-SI/gn_module_monitoring#42]
Ajouter un test de la configuration du module:

- intégrité des fichiers `json`
- test si les variables custom sont bien définies
- test si les jdd sont bien renseignés
- etc...

et faire remonter les erreurs dans la page du module (message, toaster, aide pour résoudre l'erreur...).

**Comment by Mounir-jelassi** [comment]
Bonjour,
Dans Geonature Demo 2.5 et Lors de l'ajout d'une nouvelle observation à partit de sous module de test (Monitoring) la recherche dans la liste des taxons n'aboutit pas.

**Comment by joelclems** [comment]
Effectivement, (et il y a d'autres bugs) la version de module monitoring n'a pas été mise à jour sur le site de demo depuis la sortie 2.5. j'essaye de faire ça rapidement.
